aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -402,15 +458,15 @@ sub pxelink
         $path = $1;
         $this_app->debug (5, "Local booting from $path");
     } elsif ($cmd eq RESCUE || $cmd eq LIVECD || $cmd eq FIRMWARE) {
-        $path = link_filepath($cfg, $cmd);
-        if (! -f $path) {
+        $path = link_filepath($cfg, $cmd, $variant);
+        if (! -f $path && ! $CAF::Object::NoAction) {
```

Comment:
`NoAction`? if this is purely for unittests, add a comment to say so. if you really want to make this noaction safe, use `CAF::Path` for file operations